renderer/reducers/peers.js

```diff
@@ -27,8 +27,8 @@ export function getPeers() {
 // Send IPC event for peers
 export const fetchPeers = () => async dispatch => {
   dispatch(getPeers())
-  const lightning = await lightningService
-  const peers = await lightning.listPeers()
+  const grpc = await grpcService
```

Comment:
In other similar actions, worker calls are wrapper with `try catch` blocks

Reply:
As per slack, these should be wrapped with failure cases, but since we don't already have failure cases for these actions I don't want to add to the scope of this PR by introducing them here.